Sources/SessionManager/SessionManager.swift

```diff
@@ -122,14 +119,14 @@ public class SessionManager {
     }
 
     public func invalidateSession() async throws -> Bool {
```

Comment:
I'm wondering if this function should still be boolean.

If you look at the values that can be returned from the function it is either true or an error, there is no false condition. So it may be better to be void instead.

Reply:
Can also be left as-is for now as well.